pkg/sql/prepare.go

```diff
@@ -44,6 +45,14 @@ type PreparedStatement struct {
 	ProtocolMeta interface{} // a field for protocol implementations to hang metadata off of.
 
 	memAcc WrappableMemoryAccount
+	// constantAcc handles the allocation of various constant-folded values which
+	// are generated while planning the statement.
+	constantAcc *mon.BoundAccount
```

Comment:
I'm always a little concerned that I'm going to accidentally copy a `BoundAccount` by value, so I tend to default to pointers, but I think you're right that using a value here is better, updated.